### PR TITLE
Lab2 Maxim Ksenofontov

### PR DESCRIPTION
**Pull request by e2xen**

### CODE REVIEW

File: app_php/.env [hunk 1]
```diff
@@ -0,0 +1,52 @@
+APP_NAME=Laravel
+APP_ENV=local
+APP_KEY=base64:Cb4cKZA9KoiKIury5BS5s5E3ssxeN7DutoL9DBdfVBw=
```
Comment (mcflydesigner) [inline app_php/.env:3]: I guess it is better not to expose secret keys(use env, for example)

**Comment by mcflydesigner** [comment]
Great job overall!
Approved.

File: app_php/.env (hunk 1 above)
Comment (e2xen) [reply app_php/.env:3]: Thanks, I will fix it in further versions

File: app_python/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,17 @@
+FROM python:3.8-alpine
```
Comment (Khalil19-99) [inline app_python/Dockerfile:1]: I think using slim image can be better from alpine as best practice

**Comment by Khalil19-99** [comment]
Great job overall!
Approved.

**Pull request closed without merging**